src/utils/explicitBipKeyDerivation.js

```diff
@@ -0,0 +1,56 @@
+import nacl from 'tweetnacl';
+import createHmac from 'create-hmac';
+
+if (global._bitcore) delete global._bitcore;
+const mnemonic = require('bitcore-mnemonic');
+
+const ED25519_CURVE = 'ed25519 seed';
+const HARDENED_OFFSET = 0x80000000;
+
+const getMasterKeyFromSeed = (seed) => {
```

Comment:
Utility function require proper documentation of function signatures. Please add documentation explaining:
- Function purpose in detail.
- Property type, range, options and explanation.
- Returned value type and explanation

For all the functions in this file.